platforms-serverless/firebase-functions/functions/pnpm-lock.yaml

```diff
@@ -0,0 +1,1625 @@
+lockfileVersion: 5.4
```

Comment:
A bit weird that we have a package.json and lockfile in the functions folder itself.

Reply:
Good catch, will add an exclusion (firebase does not support pnpm)

Reply:
There's only a handful of these. Such projects will have an exclusion via `.npmrc` to tell pnpm not to generate a lockfile.